CONTRIBUTING.md

```diff
@@ -179,97 +173,11 @@ kills contributor momentum.
-* Come chat with us in [Slack][slack].
-
-They must follow our [Code of Conduct](CODE_OF_CONDUCT.md).
-
-## Contributor
-
-[Contributors][contributors] have the following capabilities:
-
-* Have issues and pull requests assigned to them
-* Apply labels, milestones and projects
-* [Mark issues as duplicates](https://help.github.com/en/articles/about-duplicate-issues-and-pull-requests)
-* Close, reopen, and assign issues and pull requests
-
-They must agree to and follow this Contributing Guide.
-
-### How to become a contributor
-
-To become a contributor, the maintainers of the project would like to see you:
-
-* Comment on issues with your experiences and opinions.
-* Add your comments and reviews on pull requests.
-* Contribute pull requests.
-* Open issues with bugs, experience reports, and questions.
-
-Contributors and maintainers will do their best to watch for community members
-who may make good contributors. But don’t be shy, if you feel that this is you,
-please reach out to one or more of the contributors or maintainers.
-
-[contributors]: https://github.com/orgs/deislabs/teams/porter-contributors
-
-## Maintainer
-
-[Maintainers][maintainers] are members with extra capabilities:
-
-* Be a [Code Owner](.github/CODEOWNERS) and have reviews automatically requested.
-* Review pull requests.
-* Merge pull requests.
-
-Maintainers also have additional responsibilities beyond just merging code:
-
-* Help foster a safe and welcoming environment for all project participants.
-  This will include understanding and enforcing our [Code of Conduct](CODE_OF_CONDUCT.md).
-* Organize and promote pull request reviews, e.g. prompting community members,
-  contributors, and other maintainers to review.
-* Triage issues, e.g. adding labels, promoting discussions, finalizing decisions.
-* Help organize our development meetings, e.g. schedule, organize and
-  execute agenda.
-
-They must agree to and follow the [Reviewing Guide](REVIEWING.md).
-
-[maintainers]: https://github.com/orgs/deislabs/teams/porter-maintainers
-
-### How to become a maintainer
-
-To become a maintainer, we would like you to see you be an effective
-contributor, and show that you can do some of the things maintainers do.
-Maintainers will do their best to regularly discuss promoting contributors. But
-don’t be shy, if you feel that this is you, please reach out to one or more of
-the maintainers.
-
-## Admin
-
-[Admins][admins] are maintainers with extra responsibilities:
-
-* Create new mixin repositories
-* Manage porter-* repositories
-* Manage porter-* teams
-
-[admins]: https://github.com/orgs/deislabs/teams/porter-admins
-
-### How to become an admin
-
-It isn't expected that all maintainers will need or want to move up to admin. If
-you are a maintainer, and find yourself often asking an admin to do certain
-tasks for you and you would like to help out with administrative tasks, please
-reach out to one or more of the admins.
+[ladder]: /CONTRIBUTION-LADDER.md
```

Comment:
Does the hyphen here need to be switched out with an underscore, to match the filename?